src/handlers/express.ts

```diff
@@ -0,0 +1,21 @@
+import { ApolloServer } from 'apollo-server-express';
```

Comment:
putting this in a `handlers` directory feels kinda like a lambda. Maybe this should just be in `src/server.ts` or something?